### PR TITLE
Sils/cache

### PR DESCRIPTION
**Pull request by sils1297**

### CODE REVIEW

File: coalib/parsing/Globbing.py [hunk 1]
```diff
@@ -214,21 +215,29 @@ def fnmatch(name, patterns):
...
-        for pat in _iter_alternatives(pattern):
-            pat = os.path.expanduser(pat)
-            pat = os.path.normcase(pat)
-            match = re.compile(translate(pat)).match
-            if match(name) is not None:
-                return True
+
+    for match in _compile_pattern(patterns):
+        if match(name) is not None:
+            return True
```
Comment (mtomwing) [inline coalib/parsing/Globbing.py]: I think this could be:

```python
return any(match(name) for match in _compile_pattern(patterns))
```

Comment (sils1297) [inline coalib/parsing/Globbing.py]: oh right indeed, thanks. You aronud at the center still btw.?

Comment (mtomwing) [inline coalib/parsing/Globbing.py]: Nope. I'm back in Canada now :)

**Comment by fneu** [comment]
ack https://github.com/coala-analyzer/coala/commit/d101b5b629a8402d51052ab8c6df0f5d96ade46b

**Comment by Makman2** [comment]
ack 47309a4

**Comment by Makman2** [comment]
ack 67a9242

**Comment by Makman2** [comment]
ack a6a42a6

**Comment by Makman2** [comment]
ack 67a9242

**Comment by Makman2** [comment]
ack 1a42873

**Comment by Makman2** [comment]
ack 01d9195

**Comment by Makman2** [comment]
ack 0bb726e

**Comment by Makman2** [comment]
ack fc9cee8

**Comment by sils1297** [comment]
@rultor merge

**Pull request merged**